module/TaskManagement/test/integration/TaskManagement/ConsoleSharesRemindersProcessTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace TaskManagement;
+
+use TaskManagement\Controller\Console\SharesRemindersController;
+use PHPUnit_Framework_TestCase;
+use Guzzle\Http\Client;
+use IntegrationTest\Bootstrap;
+use Prooph\EventStore\EventStore;
+use Application\Entity\User;
+use People\Entity\Organization;
+use People\Service\OrganizationService;
+use TaskManagement\Entity\Task as EntityTask;
+use TaskManagement\Entity\Stream as EntityStream;
+use TaskManagement\Entity\TaskMember;
+use TaskManagement\Entity\Vote;
+use TaskManagement\Service\TaskService;
+use Zend\Console\Request as ConsoleRequest;
+use TaskManagement\Service\MailService;
+
+class ConsoleSharesRemindersProcessTest extends \PHPUnit_Framework_TestCase
+{
+
+	private $controller;
+	private $owner;
+	private $member;
+	private $task;
+	private $transactionManager;
+	private $taskService;
+	private $orgService;
+	private $mailService;
+
+	protected function setUp()
+	{
+		$serviceManager = Bootstrap::getServiceManager();
+		$this->mailService = $serviceManager->get('AcMailer\Service\MailService');
+
+		$this->mailcatcher = new Client('http://127.0.0.1:1080');
```

Comment:
c'è una classe MailBox che wrappa Mailcatcher, valuterei se usare quella per evitare di spargere duplicazione in giro